systems/plants/RigidBody.m

```diff
@@ -247,15 +247,13 @@ function dofnum(obj)
 
     function body = makeBelongToCollisionFilterGroup(body,collision_fg_id)
       for id = reshape(collision_fg_id,1,[])
-        body.collision_filter.belongs_to = ...
-          bitor(body.collision_filter.belongs_to,bitshift(1,id-1));
+        body.collision_filter.belongs_to(collision_fg_id) = true;
```

Comment:
I was running into bugs caused by converting MATLAB integer types into `std::bitset` objects. The logic is the same, really, it's just that now the MATLAB side properties are logical vectors. In `constructModelmex` I loop over those vectors and set the appropriate bits in the `std::bitset`.